### DISCUSSION TITLE
Some of my image folders don't show up as galleries

### DISCUSSION THREAD

**Discussion by utubeuser234** [stashapp/stash#5060] [Q&A] [answered]
I hope I am posting this in the right place. I just started using Stash. Many of my image folders don't show up as galleries. I would like them to. Please assist.

**Comment by DogmaDragon** [comment, marked as answer]
Make sure you have set Settings > Library > Gallery and Image options > Create galleries from folders containing images.

**Reply by utubeuser234** [reply]
Thanks!
